src/cli/plugin/pluginDownloader.js

```diff
@@ -73,7 +75,12 @@ module.exports = function (settings, logger) {
 
   function wrappedRequest(requestOptions) {
     return Promise.try(function () {
-      return request.get(requestOptions);
+      let urlInfo = url.parse(requestOptions.url);
+      if (urlInfo.protocol === 'file:') {
```

Comment:
`/^file/.test(urlInfo.protocol)` might be a little safer here, it's not clear from the node docs that the protocol always contains the trailing `:`